guide/matrix-vs-discord.md

```diff
@@ -74,36 +76,43 @@ Note that Matrix does not (and cannot, due to its decentralized nature) paywall
 | Specific avatars[^2] | Requires Nitro. | [**Supported**](../features/#attachments) (`/myroomavatar`). See "Attachments" for limits. |
 | 2FA | Email or SMS/TOTP. | Not required for login, but required (QR code, emoji verification, or Security Key) for viewing past encrypted messages. |
-| **Text messages** | Maximum 2000 characters. Supports Markdown (with some modifications). | **No maximum. [Supports Markdown and HTML.](../features/#text)** |
-| Attachments | Maximum 8 MB. | **Maximum 50~100 MB** (for most homeservers; customizable if you run your own homeserver). |
-| Custom emotes in messages | Free users can only use static emotes defined within the "server." | It is possible to insert user-defined static emotes in messages, see [here](../features/#attachments). No support for animated emotes. |
+| **Text messages** | Maximum 2000 characters. Supports Markdown (with some modifications). | **Up to ~65200 bytes (up to ~40770 bytes if a formatted message with plain text fallback sent).[^7] [Supports Markdown and HTML.](../features/#text)** |
+| Attachments | Maximum 8 MB (maximum 100 MB for users with Nitro, and also for all users in a "server" with boost tier 3). | **Maximum 50~100 MB** (for most homeservers; customizable if you run your own homeserver). |
+| Custom emotes in messages | Free users can only use static emotes defined within the "server". Using emojis from outside the "server" and using animated emojis are limited to Nitro users only. | It is possible to insert user-defined static emotes in messages, see [here](../features/#attachments). No support for animated emotes. |
 | Reactions | Only emotes (Unicode or custom ones) | Unicode emotes and [text](../features/#reactions). |
 | Stickers | Only stickers defined within the "server". | **Unlimited with setup.** See [here](../features/#stickers). |
-| Read receipts | Not supported. | **Supported.** |
+| Public read receipts | Not supported. | **Supported.** |
 | **Direct messages** | Not encrypted. | **Encrypted by default**, including VoIP. |
 | Starting a DM | Depending on privacy settings, initiating a DM requires the two users to have established "friendship" or have certain mutual "servers." Users are given the choice to accept, remove, or report a DM (since late 2021). | Initiating a DM solely requires the recipient to accept the request[^3]. Users can leave DMs anytime they wish. |
-| **Group chats** | A channel is associated with a "server." You can only join 100 "servers." | A room is standalone, but can be optionally included and associated with a Space. You can join **unlimited** amount of rooms. |
+| **Group chats** | A channel is associated with a "server." You can only join 100 "servers," 200 with Nitro. | A room is standalone, but can be optionally included and associated with a Space, which is just a room linking to other rooms. You can join **unlimited** amount of rooms. |
 | VoIP in groups | Supported. | Not supported until early 2022. |
```

Comment:
Technically supported but only via Jitsi Widgets and Element clients are the only clients that have proper support for this. Expected to be replaced during 2022 with a better system that is a part of the matrix spec. 
```suggestion
| VoIP in groups | Supported. | Limited Support via Integration with Jitsi. Expected to be replaced by a better solution during 2022. |
```